> What failures?

ofborg ("libbpf, libbpf.passthru.tests on x86_64-linux" and aarch64-linux both failed according to the github checks: "Unexpected error: command failed with exit code 1"

There's just so much being rebuilt that the log file is huge and I couldn't fgure what failed to build at the time, but getting the full log (.. 98MB! I only checked x86_64) here it is:
```
error: builder for '/nix/store/2y6l2hr8grmf22w96skikw97ljlh4j84-perl5.36.0-XML-LibXML-2.0207.drv' failed
 with exit code 2;
       last 10 log lines:
       >
       > Test Summary Report
       > -------------------
       > t/35huge_mode.t                                  (Wstat: 512 (exited 2) Tests: 5 Failed: 2)
       >   Failed tests:  2-3
       >   Non-zero exit status: 2
       > Files=76, Tests=2535, 14 wallclock secs ( 0.49 usr  0.11 sys +  8.36 cusr  2.72 csys = 11.68 CP
U)
       > Result: FAIL
       > Failed 1/76 test programs. 2/2535 subtests failed.
       > make: *** [Makefile:1326: test_dynamic] Error 255
```

okay, well, I'll just assume things are unrelated and consider this good. I agree changes are really minimal so there's no reason any of it fails...

--- a/pkgs/os-specific/linux/libbpf/default.nix
+++ b/pkgs/os-specific/linux/libbpf/default.nix
@@ -9,13 +9,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libbpf";
-  version = "1.2.0";
+  version = "1.2.2";
 
   src = fetchFromGitHub {
     owner = "libbpf";
     repo = "libbpf";
     rev = "v${version}";
-    sha256 = "sha256-NimK4pdYcai21hZHdP1mBX1MOlNY61iDJ+PDYwpRuVE=";
+    sha256 = "sha256-SDDdz2HKEfzHloLkb0sv5ldTo+1yJDVc9O7nj4Cjznk=";
   };
 
   nativeBuildInputs = [ pkg-config ];

